boards/cats_rev1Pro/Core/Inc/sensors/h3lis100dl.h

```diff
@@ -0,0 +1,77 @@
+// H3LIS100DL Device Library
+// Author: Luca Jost
+// 04.08.2021
+
+#pragma once
+
+#include "stm32l4xx_hal.h"
+#include "drivers/spi.h"
+#include <stdbool.h>
+
+/*	REGISTERS	*/
+#define H3LIS100DL_WHO_AM_I  0x0F  // Answer should be 0x32
+#define H3LIS100DL_CTRL_REG1 0x20
+#define H3LIS100DL_CTRL_REG2 0x21
+#define H3LIS100DL_CTRL_REG3 0x22
+#define H3LIS100DL_CTRL_REG4 0x23
+#define H3LIS100DL_CTRL_REG5 0x24
+
+#define H3LIS100DL_FILTER_RESET 0x25
+#define H3LIS100DL_REFERENCE    0x26
+#define H3LIS100DL_STATUS_REG   0x27
+
+#define H3LIS100DL_OUT_X 0x29
+#define H3LIS100DL_OUT_Y 0x2B
+#define H3LIS100DL_OUT_Z 0x2D
+
+#define H3LIS100DL_INT_CFG1 0x30
+#define H3LIS100DL_INT_SRC1 0x31
+#define H3LIS100DL_INT_THS1 0x32
+#define H3LIS100DL_INT_DUR1 0x33
+
+#define H3LIS100DL_INT_CFG2 0x34
+#define H3LIS100DL_INT_SRC2 0x35
+#define H3LIS100DL_INT_THS2 0x36
+#define H3LIS100DL_INT_DUR2 0x37
+
+#define H3LIS100DL_WHO_AM_I_CONST 0x32
+
+// CTRL_REG_1
```

Comment:
We are mixing /* */ and // comments a lot -- try to use /* */ on the global level and with longer descriptions and // only inside functions.